src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -559,9 +559,14 @@ public virtual bool IsAnonymousType
 
         /// <summary>
         /// True if the type represents a native integer. In C#, the types represented
-        /// by language keywords 'nint' and 'nuint'.
+        /// by language keywords 'nint' and 'nuint' on platforms where they were not unified
+        /// with 'System.IntPtr' and 'System.UIntPtr'.
         /// </summary>
-        internal virtual bool IsNativeIntegerType => false;
+        internal virtual bool IsNativeIntegerWrapperType => false;
```

Comment:
:memo: This internal API was just renamed. The API below will be renamed to `IsNativeInteger` after this PR is reviewed, but I wanted to leave distinct names to help with review.